src/Surfnet/StepupRa/RaBundle/Form/Type/AmendRegistrationAuthorityInformationType.php

```diff
@@ -52,7 +52,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'data_class' => 'Surfnet\StepupRa\RaBundle\Command\AmendRegistrationAuthorityInformationCommand'
+            'data_class' =>
```

Comment:
Hmm, I was under the impression we were doing this only when the RA's command deviates from the client bundle's. Maybe a brain fart. I'll put it back and add the assertions on it.